### PR TITLE
Zawy's LWMA & testnet reset

### PR DESCRIPTION
**Pull request by h4x3rotab**

### CODE REVIEW

**Comment by coinalchemist** [comment]
@h4x3rotab Great work!

**Comment by akx20000a** [comment]
@h4x3rotab Great work!

File: src/chainparams.cpp [hunk 1]
```diff
@@ -224,17 +229,20 @@ class CTestNetParams : public CChainParams {
         consensus.BIP65Height = 581885; // 00000000007f6655f22f98e72ed80d8b06dc761d5da09df0fa1dc4be4f861eb6
         consensus.BIP66Height = 330776; // 000000002104c8c45e99a8853285a3b592602a3ccde2b832481da85e9e4ba182
         consensus.BTGHeight = 1210320;
+        consensus.BTGZawyLWMAHeight = -1; // Activated on testnet
```
Comment (akx20000a) [inline src/chainparams.cpp]: Needs edit here, Block height is too large for BIP34 ~ BTG.

**Comment by akx20000a** [comment]
@h4x3rotab  Are we done?

**Comment by h4x3rotab** [comment]
The testnet is running. It looks good so far. I'm going to merge it.

**Comment by kiahmed** [comment]
@h4x3rotab whats the server and port?

**Comment by h4x3rotab** [comment]
@kiahmed DNSSeed should work. You can compile the latest binary and try to connect to the testnet automatically. Please let me know if it doesn't work. The default port is still 18338.

**Comment by kiahmed** [comment]
I can't build. Would taking the latest binary fro linux and put testnet in config file would work?

**Comment by h4x3rotab** [comment]
Unfortunately it’s impossible until we make the next release (0.14.1 or 0.16). We are working on new releases. Better to build your own client. It’s easy!

**Comment by kiahmed** [comment]
not sure why you say "impossible". Why I can't use the latest binary and manually add the test seed node?

**Comment by h4x3rotab** [comment]
@kiahmed This is a testnet reset. We have changed the consensus rule for testnet. It will end up banned by testnetv3 if you try to connect to testnet with the latest release (0.15.0.2). If you believe me I can compile a linux version for you. However, generally speaking you shouldn't trust anyone. You have to verify by yourself.

**Pull request merged**